libscript/include/pivot/script/Parser.hxx

```diff
@@ -86,5 +86,7 @@ void printComponentNode(const Node &component);
 std::string remove_comments(const std::string &line);
 /// Is line empty (or whitespace)
 bool line_is_empty(const std::string &line);
+/// Because we don't compile with boost/algorithm
```

Comment:
Nothing prevent you to add boost algorithm. We already compile with a good chunk of boost, so feel free to add algorithm